tests/AiDotNet.Tests/IntegrationTests/ConfigureMethodCoverage/README.md

```
@@ -0,0 +1,145 @@
+double featTime = TimeAction(() => featureResult.Predict(probe));
+double speedup = baselineTime / featTime;
+AssertSpeedupBetween(speedup, lowerBound: 0.8, upperBound: 10.0, "ConfigureYourMethod");
+```
+
+The 0.8× lower bound is non-negotiable — a drop-in optimization should never be slower
+than baseline; this catches the Int8Quantization 0.36× and FlashAttention 3.76× slower
+regressions.
+
+## Filter
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)